locale/templates/main.ftl

```diff
@@ -355,4 +369,6 @@ new-user-subscribe-product-updates = I'd like to receive product updates from {
 new-user-subscribe-product-assurance = We only use your email to create your account. We will never sell it to a third party.
 new-user-email-validate = Email is not valid
 new-user-email-validate-confirm = Emails do not match
-new-user-existing-account-sign-in = You already have an account, <a>Sign in</a>
+new-user-already-has-account-sign-in = You already have an account. <a>Sign in</a>
```

Comment:
Any change in the source string should result in creating a new string ID by adding `-2`. en-CA, en-GB are leveraged from en-US.